giraffa-core/src/main/java/org/apache/giraffa/hbase/BlockManagementAgent.java

```diff
@@ -375,6 +388,58 @@ private void closeBlockFile(ExtendedBlock block) throws IOException {
     }
   }
 
+  /**
+   * Calls allocateBlockFile to create a new block for the file if and only if
+   * we are modifying the Block column in HBase via this put.
+   *
+   *
+   * @param kvs
+   * @throws IOException
+   */
+  private void recoverLastBlock(List<Cell> kvs) throws IOException {
```

Comment:
Remove empty @param and @throws Javadoc. Additionally, we can add some nonnull assertions for leaseKv, blockKv, and unlocatedBlocks. Finally, line 421 does not need to split into two lines.